out/gatewayConfiguration.json

```diff
@@ -9,5 +9,7 @@
 
     "platformTrustStore": "/etc/wolkGateway/ca.crt",
 
-    "subdeviceManagement": "gateway"
+    "subdeviceManagement": "gateway",
+
+    "mqttKeepAliveSeconds": 60
```

Comment:
Emphasis that is platform connection keep alive, e.g. `platformMqttKeepAliveSeconds`, and put in after `"platformMqttUri": `